src/lm_buddy/cli/run.py

```diff
@@ -32,3 +37,10 @@ def run_finetuning(config: str) -> None:
 def run_lm_harness(config: str) -> None:
     config = LMHarnessJobConfig.from_yaml_file(config)
     lm_buddy.run_job(config)
+
+
+@group.command("prometheus", help="Run the prometheus evaluation job.")
```

Comment:
Good point! And wdyt if instead of just changing the filenames we change the CLI groups? 
E.g. `lm_buddy run eval lm-harness`, `lm_buddy run eval ragas`, etc.
Actually `lm_buddy finetune ...`, `lm_buddy eval ...` sound even better, but I am not sure whether this would break anything

Reply:
I would say if we are going to change it, we should just go with "lm-buddy finetune" and "lm-buddy evaluate ragas/prometheus/lm-harness".

But then how does the "simple" job fit in semantically? We can probably just delete that since I was just using it for initial debugging. 

No need to cram these changes into this PR, though. We can make a ticket for it.

Reply:
Agree, I think it would be good to put the simple job in a "cookbook" if we decide to separate the examples or in the documentation and take it out of the codebase

Reply:
No CLI changes in this PR, as we are planning to make broader interface changes down the line.